.github/workflows/nightly-release.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           echo "BUILD_OUTPUT_LIST=$(cat tmp2.txt | tr '\n' ',' | perl -ple 'chop')" >> $GITHUB_ENV
           rm -rf tmp.txt && rm -rf tmp2.txt
       - name: Advance nightly tag
-        uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6.4.1
```

Comment:
I suspect the exact problem is that the client changed a bit, and the current script is invalid on v6+, don't know when that changed, and since I'm not super knowledgeable on this, I figured the current implementation would be good enough as a hotfix.
But yes, we'll need a more permanent fix as I do suspect that the automation will bump it again soon enough.